services/adminService.js

```diff
@@ -0,0 +1,52 @@
+const { User, Tweet, Like, sequelize } = require('../models')
+const { Op } = require('sequelize')
```

Comment:
```suggestion
const { User, Tweet, Like, Sequelize } = require('../models')
const { Op } = Sequelize
```
可以使用這樣，因為 `models/index.js` 裡面已經有載入 `require('sequeilze')`
經過實驗結果，`Sequelize` 也可以使用 `sequelize` 裡面的方法，例如 `sequelize.literal` 或是 `sequelize.col` 等

Reply:
ok